Firmware code/smart_energy_meter/src/helpers.hpp

```diff
@@ -94,4 +94,4 @@ void urgeent() {
       fault = 0;
     }
   }
-}
+}
```

Comment:
Please don't remove trailing new lines at the end of files, this is why the linter fails.